hugo/layouts/index.html

```diff
@@ -47,12 +51,15 @@ <h2 class="pl-2 pb-2">Welcome to the ACL Anthology!</h2>
     <div class="row mb-3">
       {{ if (fileExists "/data-export/anthology.bib.gz") }}
       <div class="col-6 col-xl-12 mb-2">
-        <a class="btn btn-block btn-info" href="{{ "/anthology.bib.gz" | relURL }}">Full Anthology as BibTeX ({{ printf "%.2f MB" (div (os.Stat "/data-export/anthology.bib.gz").Size 1000000.0) }})</a>
```

Comment:
Is there a reason for these changes? I do not see the connection to the rss functionality.

Reply:
Had to retrace my steps here a bit. Got to love my own slightly cryptic commit messages: 8f2cf74266bd2dcf64ef0432f13c00011fd3f716.

After looking at the previous commit and the [documentation of `relURL`](https://gohugo.io/functions/urls/relurl/#input-begins-with-a-slash) I think it boils down to this note from the documentation:

> The last three examples are not desirable in most situations. As a best practice, never include a leading slash when using this function.